lang_java/tree_sitter/CONTRIBUTING_TREE_SITTER.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+## Overview
+
+This directory houses the files used for converting `tree-sitter` grammar to `pfff` grammar. 
+
+## Goal 
```

Comment:
I would add here why tree-sitter was selected in the first place. The slide you presented internally would be helpful!